pkg/v5/resource/chartconfig/desired.go

```diff
@@ -14,7 +14,7 @@ import (
 const (
 	chartConfigAPIVersion           = "core.giantswarm.io"
 	chartConfigKind                 = "ChartConfig"
-	chartConfigVersionBundleVersion = "0.1.0"
+	chartConfigVersionBundleVersion = "0.2.0"
```

Comment:
Why is that now? Should that also be reflected in the changelogs?

Reply:
0.2.0 of chart-operator supports setting values. I'll update the changelog.

Reply:
Done.